scripts/zones/Bastok_Markets/npcs/Harmodios.lua

```diff
@@ -48,8 +48,7 @@ entity.onEventFinish = function(player, csid, option)
         player:setCharVar("WildcatBastok", utils.mask.setBit(player:getCharVar("WildcatBastok"), 10, true))
     elseif csid == 490 then
         player:startEvent(491)
-    elseif csid == 491 then
-        player:setCharVar("comebackQueenCS", 2)
+		player:setCharVar("comebackQueenCS", 2)
```

Comment:
Align this line properly and this is good to go.

Reply:
okie thanks

Reply:
deleted tab that mysteriously got in there and replaced with spaces to line up.